src/shared/hooks/useTimeAgo.test.ts

```diff
@@ -0,0 +1,21 @@
+import { add } from 'date-fns';
+import { renderHook } from '@testing-library/react';
+
+import { mockI18Next } from '__mocks__/i18next';
+
+import { useTimeAgo } from './useTimeAgo';
+
+jest.mock('react-i18next', () => mockI18Next);
```

Comment:
It will be automatically mocked. There's no need to explicitly call jest.mock('module_name').

, https://jestjs.io/docs/manual-mocks#mocking-node-modules